app/src/main/kotlin/com/simplemobiletools/calculator/activities/MoneyActivity.kt

```diff
@@ -162,6 +164,9 @@ class MoneyActivity : SimpleActivity(), Calculator, LocationListener {
                     spawnTaxModal()
                     return
                 }
+                var taxRate: Double = (TaxOperation.getTaxRate(province)) * 100
+                var message: String = "Current Location: " + province + "\nTax rate: " + taxRate + "% "
+                Toast.makeText(applicationContext, message, Toast.LENGTH_LONG).show()
```

Comment:
Is this part of crypto conversion ?

Reply:
It is not, I have modified the commit message to link it to issue #140